OWML.Common/IModBehaviour.cs

```diff
@@ -2,11 +2,26 @@
 
 namespace OWML.Common
 {
+    /// <summary>The mod behaviour.</summary>
     public interface IModBehaviour
     {
+        /// <summary>The ModHelper attached to this mod.</summary>
         IModHelper ModHelper { get; }
+
+        /// <summary>The API attached to this mod.</summary>
+        object Api { get; }
```

Comment:
So this PR builds upon your other mod interaction PR?

Reply:
Yeah, the other PR would have to be merged first before this one